src/nvim/mbyte.c

```diff
@@ -71,6 +71,7 @@
  * some commands, like ":menutrans"
  */
 
+#include <assert.h>
```

Comment:
You are now not using assertions here, are you? At least I do not see them added.